gulpfile.babel.js

```diff
@@ -0,0 +1,33 @@
+const del = require('del');
+const eslint = require('gulp-eslint');
+const gulp = require('gulp');
+const webpack = require('webpack');
+
+const webpackConfig = require('./webpack.config.js');
+
+webpackConfig.plugins.push(new webpack.optimize.DedupePlugin());
+webpackConfig.plugins.push(new webpack.optimize.UglifyJsPlugin({
+  compress: {
+    warnings: false
+  }
+}));
```

Comment:
This is where the :magic: happens. Instead of setting these plugins directly in the ./webpack.config.js file, we inject them in during the gulp process so it doesn't impact the dev server's behavior. Or maybe it's hacky. #youdecide